servicex/models.py

```diff
@@ -26,19 +26,29 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 import hashlib
-from sqlalchemy import func, ForeignKey
+
+from sqlalchemy import func, ForeignKey, DateTime
 from flask_sqlalchemy import SQLAlchemy
 
+from servicex.db_utils import utcnow
+
 db = SQLAlchemy()
 
 
 class UserModel(db.Model):
     __tablename__ = 'users'
+    admin = db.Column(db.Boolean, default=False)
+    created_at = db.Column(DateTime, default=utcnow())
     id = db.Column(db.Integer, primary_key=True)
-    username = db.Column(db.String(120), unique=True, nullable=False)
+    email = db.Column(db.String(320), nullable=False, unique=True)
+    email_confirmed = db.Column(db.Boolean, default=False)
```

Comment:
How is `email_confirmed` used?

Reply:
It's not. Rob mentioned last week that he thinks it's unlikely we would attempt to do email confirmations manually (by that time we would probably switch to some federated identity system like Globus). Removed.